doc/classes/VisualServer.xml

```diff
@@ -1951,6 +1951,11 @@
 			<argument index="2" name="scenario" type="RID">
 			</argument>
 			<description>
+				Returns an array of object IDs intersecting with the provided 3D ray. Only visual 3D nodes are considered, such as [MeshInstance] or [DirectionalLight].
+				Use [code]instance_from_id[/code] to obtain the actual nodes.
+				A scenario RID must be provided, which is available in the [World] you want to query.
+
```

Comment:
There should not be empty lines in the XML, they will end up as empty paragraphs.